src/main/java/edu/cornell/kfs/module/purap/jaggaer/contract/xml/CustomField.java

```diff
@@ -0,0 +1,58 @@
+package edu.cornell.kfs.module.purap.jaggaer.contract.xml;
+
+import java.util.List;
+
+import jakarta.xml.bind.annotation.XmlAccessType;
+import jakarta.xml.bind.annotation.XmlAccessorType;
+import jakarta.xml.bind.annotation.XmlAttribute;
+import jakarta.xml.bind.annotation.XmlElement;
+import jakarta.xml.bind.annotation.XmlElementWrapper;
+import jakarta.xml.bind.annotation.XmlRootElement;
+import jakarta.xml.bind.annotation.XmlType;
+import jakarta.xml.bind.annotation.adapters.NormalizedStringAdapter;
+import jakarta.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "", propOrder = {
+    "internalName",
+    "customFieldValues"
+})
+@XmlRootElement(name = "CustomField")
+public class CustomField {
```

Comment:
Supplier has a similar object called CustomElement, however they contain different values. Should we consider re-naming here to avoid confusion?

Reply:
Given the results of the `Attachment` DTO discussion, I don't think this needs renaming at this time, due to the context separation in the package setup.